x-pack/plugin/eql/qa/common/src/main/java/org/elasticsearch/test/eql/EqlSpecLoader.java

```diff
@@ -55,6 +55,14 @@ private static String getTrimmedString(TomlTable table, String key) {
             spec.note(getTrimmedString(table, "note"));
             spec.description(getTrimmedString(table, "description"));
 
+            // default these values to false
+            spec.caseSensitiveOnly(Boolean.TRUE.equals(table.getBoolean("case_sensitive")));
+            spec.caseInsensitiveOnly(Boolean.TRUE.equals(table.getBoolean("case_insensitive")));
+
+            if (spec.supportsCaseInsensitive() == false && spec.supportsCaseSensitive() == false) {
```

Comment:
On the same idea of being more clear like above, I would rewrite this condition as `if (spec.isCaseSensitiveOnly && spec.isCaseInsensitiveOnly).....`.

Reply:
changed behavior in 254b5c3